src/components/panningViews/panDismissibleView.js

```diff
@@ -193,8 +210,9 @@ class PanDismissibleView extends PureComponent {
   };
 
   onInitAnimationFinished = () => {
-    this.setState({isAnimating: false});
-    this.initPositions();
+    const {forceDismissIsPending} = this.state;
+    const runAfterSetState = forceDismissIsPending ? this.animateDismiss : undefined;
+    this.initPositions({isAnimating: false, forceDismissIsPending: false}, runAfterSetState);
```

Comment:
do we really need `isAnimating` in state

Reply:
We need it in the state (but I removed `shouldDismissAfterReset` from the state)